python/python/lance/util.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (c) 2023. Lance Developers
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import datetime, timezone
+from typing import Union
+
+import pandas as pd
+
+
+def sanitize_ts(ts: Union[datetime, str]) -> datetime:
```

Comment:
does `pd.to_datetime(ts).to_pydatetime()` works as well? If so that might be more concise?